nifi-nar-bundles/nifi-riemann-bundle/nifi-riemann-nar/pom.xml

```diff
@@ -1,34 +1,40 @@
 <!--
-  Licensed to the Apache Software Foundation (ASF) under one or more
-  contributor license agreements.  See the NOTICE file distributed with
-  this work for additional information regarding copyright ownership.
-  The ASF licenses this file to You under the Apache License, Version 2.0
-  (the "License"); you may not use this file except in compliance with
-  the License.  You may obtain a copy of the License at
-      http://www.apache.org/licenses/LICENSE-2.0
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License.
--->
+      Licensed to the Apache Software Foundation (ASF) under one or more
+      contributor license agreements.  See the NOTICE file distributed with
+      this work for additional information regarding copyright ownership.
+      The ASF licenses this file to You under the Apache License, Version 2.0
+      (the "License"); you may not use this file except in compliance with
+      the License.  You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <parent>
-        <artifactId>nifi-riemann-bundle</artifactId>
-        <groupId>org.apache.nifi</groupId>
-        <version>0.4.2-SNAPSHOT</version>
-    </parent>
+  <parent>
+    <artifactId>nifi-riemann-bundle</artifactId>
+    <groupId>org.apache.nifi</groupId>
+    <version>0.4.2-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>nifi-riemann-nar</artifactId>
-    <packaging>nar</packaging>
-    <dependencies>
-        <dependency>
-            <groupId>org.apache.nifi</groupId>
-            <artifactId>nifi-riemann-processors</artifactId>
-        </dependency>
-    </dependencies>
+  <artifactId>nifi-riemann-nar</artifactId>
+  <packaging>nar</packaging>
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.nifi</groupId>
+      <artifactId>nifi-riemann-processors</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.nifi</groupId>
+      <artifactId>nifi-riemann-reporting-task</artifactId>
+      <version>${project.version}</version>
```

Comment:
Strong recommend not using ${project.version}.  Has been unreliable in the release process in the past.  Recommend simply using the 0.4.2-SNAPSHOT.  These get handled seamlessly during the release cycle.  Also recommend not declaring the versions here but rather in the bundle/pom.xml in a dependency management section and here simply pulling that dependency but without version.